Make buildbox-casd log level configurable

nanonyme opened the pull request:
Default is error

nanonyme committed:
Make buildbox-casd log level configurable

Default is error

jjardon commented on defaults/main.yml:
```diff
@@ -31,12 +31,15 @@ casd_metrics_args: >-
 casd_cache_mnt: "/srv"
 casd_certs_mnt: "/certs"
 casd_cmd: >-
   --verbose --bind {{ casd_bind }}
```
@nanonyme should we remove the "--verbose" from here?

nanonyme replied:
Definitely. Looks like I messed up with web UI. Unless someone beats me to it, I will probably fix in the morning.

nanonyme replied:
The point is log level needs to be configurable without touching this repo.

sdclarke replied:
I removed the hardcoded `--verbose` flag.

sdclarke committed:
Remove hardcoded `--verbose` flag

Additionally update the README with information about the new
`casd_log_level` variable
